pkg/server/apiextensions.go

```diff
@@ -162,7 +168,11 @@ func (p *systemCRDProvider) Keys(clusterName logicalcluster.Name) sets.String {
 			return sets.NewString()
 		}
 
-		switch clusterWorkspace.Spec.Type {
+		if clusterWorkspace.Spec.Type.Path != "root" {
```

Comment:
What is this change doing?

Reply:
Ah, good catch - we may not need this. This code seems to inject/project ClusterWorkspaceTypes, but for the system ones we don't really need to do this into child/derivative workspaces, since the user can just refer to the ones in the root for their needs.

Reply:
also for non-tests, we have `tenancyv1alpha1.RootCluster` as constant.